apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -49,68 +47,80 @@ function WpcomNux() {
 
 	const dismissWpcomNux = () => setWpcomNuxStatus( { isNuxEnabled: false } );
 
+	// @TODO: Change to 'Welcome to your new website' for Gutenboarding, in other situations keep "editor"
+	const welcomeHeading = __( 'Welcome to the WordPress editor' );
+
 	/**
 	 * Currently, the Guide content is mostly copied from Gutenberg. This can be
 	 * updated if/as we have new designs for the NUX on wpcom.
 	 */
 	return (
 		<Guide
-			className="edit-post-welcome-guide"
-			contentLabel={ __( 'Welcome to the WordPress editor' ) }
+			className="wpcom-block-editor-nux"
+			contentLabel={ welcomeHeading }
 			finishButtonText={ __( 'Get started' ) }
 			onFinish={ dismissWpcomNux }
 		>
-			<GuidePage className="edit-post-welcome-guide__page">
-				<h1 className="edit-post-welcome-guide__heading">
-					{ __( 'Welcome to the WordPress editor' ) }
-				</h1>
-				<CanvasImage className="edit-post-welcome-guide__image" />
-				<p className="edit-post-welcome-guide__text">
-					{ __(
-						'In the WordPress editor, each paragraph, image, or video is presented as a distinct “block” of content.'
-					) }
+			<GuidePage className="wpcom-block-editor-nux__page">
+				<h1 className="wpcom-block-editor-nux__heading">{ welcomeHeading }</h1>
+				<p className="wpcom-block-editor-nux__text">
+					{ __( 'Create and edit site pages, and customize the look and feel of each page.' ) }
 				</p>
+				<img
+					alt=""
+					aria-hidden="true"
```

Comment:
Since these images are all just to visualize the text, I figured we can hide them from screen-readers.